signer/serverless.yml

```diff
@@ -110,3 +110,40 @@ functions:
               - X-Amz-Security-Token
               - X-Amz-User-Agent
               - x-requested-with
+      - http:
+          method: POST
+          path: /state-service
+          authorizer:
```

Comment:
should api be signer/state-service  , signer/mpe-claim

Reply:
yes